### ISSUE TITLE
Contributing Tensorflow/Keras code

### ISSUE THREAD

**Issue by chaudharyachint08** [rasbt/cyclemoid-pytorch#1]
It is to my surprise how this works well looking complex at the first place (unlike Swish), and I will be waiting for what mathematical challenges and improvements it will bring over ReLU

<pre>
import math as m
import tensorflow as tf

def cyclemoid(x):
  'Implementation tweaked from PyTorch implementation, to be used as functions & in Lambda layers'
  pi = tf.constant(m.pi)
  term1 = tf.math.tanh(pi*x)
  term2 = tf.math.tanh(pi*tf.square(x)-0.95)**2
  term3 = tf.math.cos(tf.clip_by_value(x, clip_value_min=-3., clip_value_max=3.))**2 
  return term1 * term2 * term3

class CycleMoid(tf.keras.layers.Activation):
  'Creating dedicated class using sub-classing'
  def __init__(self, **kwargs):
    super().__init__(cyclemoid, **kwargs)

'Crafting for the easiest use'
tf.keras.utils.get_custom_objects()['cyclemoid'] = CycleMoid
</pre>

**Comment by chaudharyachint08** [comment]
Tweaking my favorite [Keras Code Example](https://keras.io/examples/vision/mnist_convnet/), below is only section that needs to be updated in notebook in the "keras-way"

<pre>
#### Change in the model definition on top of above notebook ####
model = keras.Sequential(
    [
        keras.Input(shape=input_shape),
        layers.Conv2D(32, kernel_size=(3, 3), activation="cyclemoid"),
        layers.MaxPooling2D(pool_size=(2, 2)),
        # layers.Conv2D(64, kernel_size=(3, 3), activation="cyclemoid"), # This works well, as one line substitute for below two lines
        layers.Conv2D(64, kernel_size=(3, 3), activation=None),
        CycleMoid(),
        layers.MaxPooling2D(pool_size=(2, 2)),
        layers.Flatten(),
        layers.Dropout(0.5),
        layers.Dense(num_classes, activation="softmax"),
    ]
)
</pre>

**Comment by rasbt** [comment]
Thanks! Just added it via #2
